app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
```

Comment:
It is a more conventional way to leave no arguments on the same lines with opening and closing parenthesis in the `__inti__`, place each argument on a new line.